### DISCUSSION TITLE
English to Hindi translation not working

### DISCUSSION THREAD

**Discussion by glenbhermon** [openai/whisper#1582] [General]
Here is my code 

import whisper
model = whisper.load_model("large")
result = model.transcribe("sou_eng.wav",task = "translate", language = "hindi")
print(result["text"])

Output:
Hi, Hello, My name is Suv, I am from Agartala and currently working here in Delhi. Where are you from?

But it should translate like the following 
हाय, हैलो, मेरा नाम सुवी है, मैं अगरतला से हूँ और वर्तमान में दिल्ली में काम कर रहा हूँ। आप कहाँ के निवासी हैं?

**Comment by glangford** [comment]
whisper cannot translate from English to a non-English language. The `translate` option is used to translate non-English to English.

**Reply by amanksk** [reply]
Is there any other way to do it ?

**Comment by masteryodaa** [comment]
> Here is my code
> 
> import whisper model = whisper.load_model("large") result = model.transcribe("sou_eng.wav",task = "translate", language = "hindi") print(result["text"])
> 
> Output: Hi, Hello, My name is Suv, I am from Agartala and currently working here in Delhi. Where are you from?
> 
> But it should translate like the following हाय, हैलो, मेरा नाम सुवी है, मैं अगरतला से हूँ और वर्तमान में दिल्ली में काम कर रहा हूँ। आप कहाँ के निवासी हैं?

did you find any solution for this??

**Comment by apoorvpandey0** [comment]
I'm using another model to translate the text which I get from whisper

**Reply by JackieLuffyDesign** [reply]
Could you share which model you use? Thanks.
